src/main/kotlin/no/nav/familie/ef/sak/api/gui/VedtaksbrevController.kt

```diff
@@ -15,10 +16,10 @@ import java.util.*
 class VedtaksbrevController(private val brevService: VedtaksbrevService,
                             private val tilgangService: TilgangService) {
 
-    @PostMapping("/{behandlingId}")
-    fun forhåndsvisBrev(@PathVariable behandlingId: UUID): Ressurs<ByteArray> {
+    @PostMapping("/{behandlingId}/{brevMal}")
+    fun forhåndsvisBrevV2(@PathVariable behandlingId: UUID, @PathVariable brevMal: String, @RequestBody utfylltBrev: String): Ressurs<ByteArray> {
         tilgangService.validerTilgangTilBehandling(behandlingId)
-        val respons = brevService.forhåndsvisBrev(behandlingId)
+        val respons = brevService.forhåndsvisBrev(behandlingId, objectMapper.readTree(utfylltBrev), brevMal)
```

Comment:
@blommish @charliemidtlyng Har dere noen inspill til `objectmapper.readTree(...)`?

Hvis vi bare sender en streng til `familie-brev` ble det lagt på et ekstra par fnutter ("") i `BrevClient` som gjorde vår json ugyldig.

Reply:
utfyltBrev er såpass dynamisk at det gir ikke mening å ha en domenemodell for denne? I så fall er jo readTree sannsynligvis riktig og fornuftig mtp at du validerer at det er gyldig json samtidig.

Reply:
Ja, det stemmer.